recipes/webpacker.rb

```diff
@@ -0,0 +1,43 @@
+module Recipes
+  class Webpacker < Base
+
+    def gems
+      @template.gem 'webpacker', '~> 3.0'
+    end
+
+    def init_file
+      @template.run 'rails webpacker:install'
+      @template.run "yarn add #{package_command_resources}"
```

Comment:
I don't have yarn installed. Maybe we should install it before in the script, or document it at the README?